vmm/src/vm.rs

```diff
@@ -1119,6 +1119,17 @@ impl Vm {
 
         self.configure_system(entry_point)?;
 
+        // configure_system will setup all region in memory_manager to
+        // guest system as normal memory.
+        // But virtio-mem region should be not available before virtio-mem
+        // setup.
+        // So setup virtio-mem region after configure_system.
+        self.memory_manager
+            .lock()
+            .unwrap()
+            .virtiomem_setup()
+            .map_err(Error::MemoryManager)?;
+
```

Comment:
Thanks for your great fix. this patch can also fix the virtiofsd crash issue about using virito-mem and virtio-fs at the same time. Since virtio-mem use case never adds a new region as the whole hotpluggable memory has already been allocated at boot time., so after virito-mem resized, the vhost user can not update about new added region.